Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/ValueStringFilter.h

```diff
@@ -18,19 +18,19 @@ namespace AZ::DocumentPropertyEditor
         ValueStringFilter();
         void SetFilterString(AZStd::string filterString);
 
+    protected:
+        // pure virtual overrides
```

Comment:
minor: perhaps change the comment to say where the overrides are from, `RowFilterAdapter`